### PR TITLE
Add library to the image url

### PR DESCRIPTION
**Pull request by moshe-kabala**

### CODE REVIEW

File: charts/tensorleap/values.yaml [hunk 1]
```diff
@@ -47,3 +47,6 @@ keycloak:
     persistence:
       enabled: true
       existingClaim: "keycloak-postgresql-data"
+  pgchecker:
```
Comment (asafyehezkel) [inline charts/tensorleap/values.yaml:50]: what is pgchecker?

File: images.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 docker.elastic.co/eck/eck-operator:2.8.0
 docker.elastic.co/elasticsearch/elasticsearch:8.10.1
 docker.io/bitnami/postgresql:11.14.0-debian-10-r28
-docker.io/busybox:1.32
+docker.io/library/busybox:1.32
```
Comment (asafyehezkel) [inline images.txt:4]: the `library` being added automatically right?

**Review by asafyehezkel — APPROVED** [review]
Well done

File: images.txt (hunk 2 above)
Comment (moshe-kabala) [reply images.txt:4]: I run the update-images - so yes

**Pull request merged**